src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -300,6 +301,7 @@ class ActivityFeed extends React.Component<Props, State> {
     } = this.props;
 
     const navigateToContact = partial(navigation.navigate, CONTACT, { contact: notification });
+    const itemStatusIcon = notificationStatus === 'pending' ? 'pending' : '';
```

Comment:
we have Pending constant for tx status, let's use it here

Reply:
Didn't notice – was reusing the code, will surely update, thanks 👍